assets/js/src/core/components/editor-tabs/editor-tabs.tsx

```diff
@@ -30,11 +31,27 @@ export interface IEditorTabsProps {
   showLabelIfActive?: boolean
 }
 
+export interface IconWrapperProps {
```

Comment:
I think this interface can be removed? Seems like it was moved to the tooltip component already? :)

Reply:
Absolutely!